Cython/Utils.py

```diff
@@ -29,6 +29,9 @@
 
 PACKAGE_FILES = ("__init__.py", "__init__.pyc", "__init__.pyx", "__init__.pxd")
 
+_CACHE_METHOD_NAME = "__{0}_cache"
+_METHOD_CACHE_PATTERN = re.compile(r"^__(.+)_cache$")
```

Comment:
```suggestion
_CACHE_METHOD_NAME = "__{0}_cache"
_CACHE_METHOD_PATTERN = re.compile(r"^__(.+)_cache$")
```

Reply:
What is the meaning of the change?
This is the method cache, if you want to change the order, then the cached method. Not "cache method".

Reply:
Both deal with exactly the same thing; one is the name, one is the pattern for catching that name. Therefore the two variable names should coincide, but whether `METHOD_CACHE_{NAME,PATTERN}` or `CACHE_METHOD_{NAME,PATTERN}` is not important IMO.

Reply:
Ok, these names really cause confusion, I will change them to more descriptive ones.

.. probably for `_CACHE_NAME` and `_CACHE_NAME_PATTERN`

Reply:
I think I'll change them as we come to some kind of agreement, there is no point in making a bunch of commits.

Reply:
If we want to specify what kind of cache this is, then we can use "method cache" instead of "cache".

Reply:
So what @h-vetinari?
You are ok with `_CACHE_NAME` and `_CACHE_NAME_PATTERN`
or is `_METHOD_CACHE_NAME` and `_METHOD_CACHE_NAME_PATTERN` better?

or something else?

Reply:
I'm not calling any shots here, but I just wanted to note that the variables describing the same thing should be named consistently. Both your most recent suggestions sound OK to me from that POV.

Reply:
Ok, I'll stick up with _CACHE_NAME

Reply:
[resolved](https://github.com/cython/cython/pull/4338/commits/e44573cec82b2d6d97ed8eff868590cd23737675)